hw/ip/otbn/dv/model/otbn_model.cc

```diff
@@ -306,6 +306,14 @@ void OtbnModel::edn_urnd_step(svLogicVecVal *edn_urnd_data /* logic [31:0] */) {
   iss->edn_urnd_step(edn_urnd_data->aval);
 }
 
+void OtbnModel::set_keymgr_value(svLogicVecVal *key0 /* logic [31:0] */,
+                                 svLogicVecVal *key1 /* logic [31:0] */,
+                                 bool valid) {
+  ISSWrapper *iss = ensure_wrapper();
+
+  iss->set_keymgr_value(key0->aval, key1->aval, valid);
```

Comment:
The `aval` thing here isn't going to work (because you're supposed to have more than 32 bits of data). You'll probably have to extract the `aval` fields in a for loop.

OR: Probably easier: Since we don't really support 4-value logic in the ISS anyway, why not change the DPI signature to take `bit [255:0]` instead and add an is-known assertion in the SystemVerilog code. Then the C++ will be much easier and it will work in exactly the same circumstances.